### PR TITLE
Update to meeting time

### PR DESCRIPTION
**Pull request by bridgetkromhout**
Meeting time has changed.

Signed-off-by: Bridget Kromhout <bridget@kromhout.org>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ like +NetworkPolicy+, +Ingress+ and +CustomMetrics+.
 
 ### Community Meeting
 
-* Community Meeting: every other Wednesday at 9:30-10:30 Pacific: [https://zoom.us/j/448032371](https://zoom.us/j/448032371)
+* Community Meeting: every other Wednesday at 10:00-10:30 Pacific: [https://zoom.us/j/448032371](https://zoom.us/j/448032371)
```
Comment (stefanprodan) [inline README.md]: My understand was that the meeting start time stays the same but it's 30m shorter

Comment (bridgetkromhout) [reply README.md]: From the March 18th [meeting minutes](https://docs.google.com/document/d/1NTBaJf6LhUBlF8_lfvBBt_MbyPvT-6CZNg6Ckpm_yCo/edit#):
```
- Adjust meeting time - could it start 30min later? (Michael)
  - We’re going to try just using the second half of this timeslot
```

Comment (lachie83) [reply README.md]: @mhausenblas ^^^

Comment (mhausenblas) [reply README.md]: Yup Stefan has it correct as far as I understood the consensus. I would, however, suggest to send out a mail to the list (by provisional chair?) to confirm and give everyone, especially folks who were not able to join, a chance to comment?

Comment (mhausenblas) [reply README.md]: I'm sorry, I meant to say that Lachie has it correct, starting at 10am PST ... I blame the new native GitHub client for the SNAFU 8-}

**Comment by bridgetkromhout** [comment]
> I would, however, suggest to send out a mail to the list (by provisional chair?) to confirm and give everyone, especially folks who were not able to join, a chance to comment?

Hi, @mhausenblas - I updated the invite so everyone should have a calendar event with the shorter meeting starting 30min later. I also added this as a discussion item for the next meeting. Thanks!

**Pull request merged**